dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -221,7 +221,7 @@ public AbstractInterfaceConfig(ModuleModel moduleModel) {
     /**
      * The url of the reference service
      */
-    protected final transient List<URL> urls = new ArrayList<URL>();
+    protected final transient List<URL> urls = new ArrayList<>();
```

Comment:
**Scalability Issue**: Usage of raw type ArrayList for URLs collection compromises type safety. In a highly scalable system, ensuring type safety is crucial for maintaining data integrity and system stability under load. <br> **Fix**: Change the declaration from raw type to a parameterized type to ensure type safety. For example, use 'List<URL>' instead of just 'List'. <br> **Code Suggestion**: 
 ```
 -    protected final transient List<URL> urls = new ArrayList<URL>();
 +    protected final transient List<URL> urls = new ArrayList<>();
 ```